packages/ui/fixtures/dashboard.json

```diff
@@ -0,0 +1,92 @@
+[
```

Comment:
neat! so this would eventually come from the database

Reply:
ya